build.RPi.sh

```diff
@@ -0,0 +1,47 @@
+#!/bin/bash
+SIGNALRCLIENT_BUILD_DIR=build.$AC_PLATFORM
+if [[ ! -d ${SIGNALRCLIENT_BUILD_DIR} ]]; then
+    mkdir ${SIGNALRCLIENT_BUILD_DIR}
+fi
+cd ${SIGNALRCLIENT_BUILD_DIR}
+rm -rf *
+
+
+PLATFORM_ROOT=$SIGNALR_ROOT/.build_RPi
+BOOST_ROOT=${PLATFORM_ROOT}/boost-1.61-RPi
+CPPRESDSDK_ROOT=${PLATFORM_ROOT}/accessapi-artifacts-RPi-ms/cpprest
+OPENSSL_ROOT=${PLATFORM_ROOT}/openssl-1.0.2l-RPi
+
+command="cmake .. -DCMAKE_TOOLCHAIN_FILE=$HOME/projects/.launchers/automation/build/RPi/RPi.toolchain  \
```

Comment:
Please use the path "/opt/RPi/RPi.toolchain" instread of "$HOME/projects/.launchers/automation/build/RPi/RPi.toolchain". I guess /opt/ is the actual directory for all toolchains.

Reply:
Changed accordingly.